luigi/contrib/hadoop_jar.py

```diff
@@ -64,23 +64,40 @@ def __init__(self):
         pass
 
     def run_job(self, job):
+        ssh_config = job.ssh()
+        if ssh_config:
+            host = ssh_config.get("host", None)
+            key_file = ssh_config.get("key_file", None)
+            username = ssh_config.get("username", None)
```

Comment:
In the documentation you mentioned these to be required fields. So please don't use `get()` and silently ignore missing fields. Instead use `[]` to make sure they exist.